docs/tutorials/jokes.md

```diff
@@ -2626,6 +2627,7 @@ export const action: ActionFunction = async ({
 export default function Login() {
   const actionData = useActionData<ActionData>();
   const [searchParams] = useSearchParams();
+
```

Comment:
Any reason for all the extra lines? ESLint shouldn't have an effect on formatting.